opentaxii.persistence.sqldb.SQLDatabaseAPI.get_service() incorrectly calls Service.get() instead of Service.query.get()

Found this when a unit test of mine failed with AttributeError:

<pre>
ret_se = SQLDatabaseAPI.get_service(self, service_id)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = &lt;(snipped) object at 0x7fe80d007ad0&gt;, service_id = 'test_cm_svc-A'

    def get_service(self, service_id):
>       return conv.to_service_entity(Service.get(service_id))
E       AttributeError: type object 'Service' has no attribute 'get'

../../../../../../../local/lib/python2.7/site-packages/opentaxii/persistence/sqldb/api.py:57: AttributeError
================================================================================================================== 1 failed, 3 passed in 0.53 seconds ==================================================================================================================
</pre>

--- a/opentaxii/persistence/sqldb/api.py
+++ b/opentaxii/persistence/sqldb/api.py
@@ -54,7 +54,7 @@ class SQLDatabaseAPI(OpenTAXIIPersistenceAPI):
         return [conv.to_service_entity(s) for s in services]
 
     def get_service(self, service_id):
-        return conv.to_service_entity(Service.get(service_id))
+        return conv.to_service_entity(Service.query.get(service_id))
 
     def update_service(self, obj):
 

